tests/doxygen/usb-logging.dox

```diff
@@ -0,0 +1,23 @@
+/**
+ * @page usb_logger USB Logging
+ * @brief Accessing the serial console.
+ *
+ * [TOC]
+ *
+ * @section usb_Overview Overview
+ *
+ * Ja Rule devices provide a serial console over the USB port. This allows a
+ * user to connect using a <a
+ * href="https://en.wikipedia.org/wiki/Terminal_emulation">terminal
+ * emulator</a>, such as <a
+ * href="http://linux.die.net/man/1/minicom">minicom</a> (Unix) or Hyperterm
+ * (Windows).
+ *
+ * The settings are 8N1, at 9600.
```

Comment:
Add the word baud, possibly explain 8N1, or do we think people will have that specialised knowledge?